File: prob6/prob6.js
var randPrint=function(randomNumber){
  var list=[]
  if(randomNumber > 40){
    for(var i=40;i<randomNumber;i++){
          if (i % 2 !==0){
            list.push(i);
          }
    }
    return list.reverse();
  }

  else{
    for(var i=0;i<randomNumber;i++){
          if (i % 2 !==0){
            list.push(i);
          }
    }
    return list.reverse();
  }


};

var randomNumber=(Math.floor(Math.random()*100));

console.log("the number is "+randomNumber);
console.log("the list" + randPrint(randomNumber));

File: prob6/prob6.py
from random import randrange



def generateList(num):
    arrayy=[]
    array=[]
    if num > 40:
        for i in range(40, num+1):
            if i % 2 !=0:
                arrayy.append(i)
        return arrayy
    else:
        for i in range(0,num+1):
            if i % 2 !=0:
                array.append(i)
        return array

def reverse(lst):
    reverse_list = []
    for i in reversed(lst):
        reverse_list.append(i)
    return reverse_list

numb = randrange(100)
print("number is {0}".format(numb))
print(reverse(generateList(numb)))
